### PR TITLE
Add: Score-board-page

### PR DESCRIPTION
**Pull request by gauravsinhaweb**

### CODE REVIEW

**Review by JuvitaSaini09 — APPROVED** [review]
good to merge

File: pages/dashboard.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
```
Comment (PuneetP16) [inline pages/dashboard.html]: UX Suggestion: On hover over the close button changing the cursor to a pointer would be preferable as per UX concerns.
image

**Review by PuneetP16 — APPROVED** [review]
Code looks clean and concise moreover the effect on image hover looks fantastic, good work.
Go ahead and merge!

File: style/dashboard.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+.wrapper_card > .card {
+  width: auto;
+}
+.wrapper-main {
+  margin: 2rem;
+}
+.showcase {
+  color: #f0b106;
```
Comment (pankaj-prog) [inline style/dashboard.css:8]: use color variables or put a comment here

**Review by j24m — APPROVED** [review]
Code seems fine, but UI is really amazing. Really good work. :)

File: pages/dashboard.html (hunk 1 above)
Comment (gauravsinhaweb) [reply pages/dashboard.html]: I'll do It later, I think It's a minor change so skipping it for now. Thanks!

File: style/dashboard.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  color: var(--text-primary);
+  padding: 1rem 2rem;
+  align-items: baseline;
+  margin: 1.5rem 0;
+  user-select: none;
+}
+.card_score span:nth-child(2),
+.card_score span:nth-child(3) {
+  font-size: 1.2rem;
+}
```
Comment (pankaj-prog) [inline style/dashboard.css]: this is good that you didn't repeat this

**Review by pankaj-prog — APPROVED** [review]
try reducing the gap between the elements, UI may look better. 
give spacing between watch icon and time.

Rest, the way you have used utility classes is good

**Pull request closed without merging**